build/installer_linux.sh

```diff
@@ -316,8 +316,16 @@ echo "[+] Extracting archive..."
 FIRST_ARCHIVE=${ARCHIVE_NAME_LIST[0]}
 "${COMMAND_7Z}" x "${FIRST_ARCHIVE}" -y
 
-# Get AppImage filename from 7z archive
-APPIMAGE=$("${COMMAND_7Z}" l -slt -ba "${FIRST_ARCHIVE}" | grep 'Path = ' | head -n1 | sed 's/Path = \(.*\)/\1/')
+# Rename
+APPIMAGE="VOICEVOX.AppImage"
+OUT_APPIMAGE_NAME=$("${COMMAND_7Z}" l -slt -ba "${FIRST_ARCHIVE}" | grep 'Path = ' | head -n1 | sed 's/Path = \(.*\)/\1/')
+if [ "$OUT_APPIMAGE_NAME" != "$APPIMAGE" ]; then
+    if ! mv "$OUT_APPIMAGE_NAME" "$APPIMAGE"; then
+        echo "Failed rename a AppImage file."
+        rm "$OUT_APPIMAGE_NAME"
+        exit 1
+    fi
+fi
```

Comment:
* コマンド失敗時のエラーハンドリングはここ以外になく、かつファイルデリートもちょっと中途半端かも？
	* 例えばクリーンするのであればダウンロードしてきたファイルの削除も必要かも
* `OUT_APPIMAGE_NAME`だと出力されたappimageの名前なのか、出力するappimageの名前なのか若干わからないかも

ということでこんな感じにするのとかどうでしょう！！
```suggestion
EXTRACTED_APPIMAGE_NAME=$("${COMMAND_7Z}" l -slt -ba "${FIRST_ARCHIVE}" | grep 'Path = ' | head -n1 | sed 's/Path = \(.*\)/\1/')
if [ "$EXTRACTED_APPIMAGE_NAME" != "$APPIMAGE" ]; then
    mv "$EXTRACTED_APPIMAGE_NAME" "$APPIMAGE"
fi
```

Reply:
こちらに変更しました。